Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: up build down dependencies test
+
+down:
+	docker-compose down
+
+build:
+	docker-compose build
+
+up: build
+	docker-compose up -d
+
+dependencies:
+	pip install --upgrade pip
+	pip install -r tests/requirements.txt
+
+test: dependencies up
```

Comment:
Could we wrap this in tox so that dependencies are in a virtualenv and functional test and we can then call our additional tests such as linting and unit in the same way?

Reply:
Sure. Lets do that in the next PR. I don't want it to block this PR from been merged.